aten/src/ATen/native/native_functions.yaml

```diff
@@ -2686,10 +2686,10 @@
     SparseCPU: sparse_sparse_matmul_cpu
     SparseCUDA: sparse_sparse_matmul_cuda
 
-- func: _sparse_matrix_mask_helper(Tensor t, Tensor mask_indices) -> Tensor
+- func: _sparse_mask_helper(Tensor t, Tensor mask_indices) -> Tensor
```

Comment:
We should take this out of native_functions.yaml. @ngimel is going to write a DeveloperFAQ entry into how to support CPU and CUDA dispatch without a native_functions.yaml entry, @IvanYashchuk and @kurtamohler also did this recently, we think, and may be able to explain the process.

Reply:
You can grep for `REGISTER_DISPATCH`, `DEFINE_DISPATCH`, and `DECLARE_DISPATCH` to see how to create a CPU/CUDA dispatch function. Usually (if not always), these functions are named `*_stub`. For instance, `eig_out` calls `eig_stub` to dispatch to the device-specific implementation.

Reply:
I would like to do this in another PR. I tried to implement in that way in the native/sparse module and it doesn't work the the code is in this folder. There is a TODO, and I guess is the origin of the problem. https://github.com/pytorch/pytorch/blob/71ca600af93b4ad58ca6fc083fa5d42a56b43194/aten/src/ATen/native/DispatchStub.h#L37